docs/source/troubleshooting.rst

```diff
@@ -0,0 +1,43 @@
+.. _troubleshooting:
+
+Troubleshooting
+===============
+
+If you're having any trouble running your Pyquil programs locally or on the QPU, please check the
+following things before sending a support request. It will save you time and make it easier for us
+to help!
+
+1. Ensure that your pyQuil version is up to date. If you're using ``pip``, you can do this with
+   ``pip freeze``. Within your script, you can use ``__version__``:
+
+   .. code:: python
+
+    import pyquil
+    print(pyquil.__version__)
+
+   You can update pyQuil with ``pip`` using ``pip install pyquil --upgrade``. You can find
+   the latest version available at
+   `our releases page <https://github.com/rigetti/pyquil/releases>`_ or
+   `on PyPi <https://pypi.org/project/pyquil/>`_.
+
+2. If the error appears to be authentication-related, or makes any mention of your
+   ``user_auth_token``, then please update your token following the directions at
+   https://qcs.rigetti.com/auth/token.
+
+3. Run your script with debug logging enabled. If you're running a script, you can enable that
+   using an environment variable:
+
+   .. code::
+
+    LOG_LEVEL=DEBUG pyquil my_script.py
+
+   .. code:: python
+
+    import logging
+    from pyquil.api._logger import logger
```

Comment:
Pointing users to a private module is less-than-ideal, but I'd rather have this out and then formalize our API, rather than continue to mask this important debugging tool. I'm not confident that `pyquil.api._logger` should stay the way I wrote it a few months ago, so it's not as easy as just adding an import to `pyquil.api.__init__.py`.

Reply:
I think it is fine for this -- we are really only suggesting that they use it in the context of troubleshooting, after all.